src/main/java/it/cnr/isti/hpc/wikipedia/parser/ArticleParser.java

```diff
@@ -519,15 +521,17 @@ private void setLists(Article article, ParsedPage page) {
 	private void setDisambiguation(Article a) {
 
 		for (String disambiguation : locale.getDisambigutionIdentifiers()) {
-			if (StringUtils.containsIgnoreCase(a.getTitle(), disambiguation)) {
+			if (StringUtils.containsIgnoreCase(a.getTitle(), "(" + disambiguation + ")")) {
```

Comment:
I'm just wondering whether `disambiguation` might either contain some whitespaces and/or parenthesis. Should we do some form of trimming / normalising ?